src/javaRosa.js

```diff
@@ -792,7 +792,12 @@ define([
     };
 
     var itextMediaBlock = function (mug, options) {
-        var block = itextConfigurableBlock(mug, options);
+        var block = itextConfigurableBlock(mug, options),
+            pathPrefix = options.pathPrefix;
+
+        if(!_.isString(options.pathPrefix)) {
```

Comment:
Ha. I can see how that is an issue.

When I thought about it, I looked at it the other way. 

If `options.pathPrefix = true` then the upload path is `jr://blah/en/true/data/questionid`, which is a completely valid path and since a basic user would not look at any uri they would have no idea there was a bug (since it isn't really one), then later when we see a bug report we have no idea where `true` is coming from.